commands/command.cpp

```diff
@@ -372,4 +372,9 @@ void removePictures(const std::vector<PictureListForDeletion> &pictures)
 	execute(new RemovePictures(pictures));
 }
 
+void addPictures(const std::vector<PictureListForAddition> &pictures)
```

Comment:
commenting on the commit message. "make dive adding undoable" and the language about dive sites... doesn't seem to match what you are doing in this commit?